azure_ad_admin/plugin.spec.yaml

```diff
@@ -155,6 +154,144 @@ types:
       type: boolean
       description: "Mail enabled"
       required: false
+  geo_coordinates:
+    altitude:
+      title: Altitude
+      description: The altitude (height), in feet, above sea level
+      type: string
+      required: false
+    latitude:
+      title: Latitude
+      description: The latitude, in decimal
+      type: string
+      required: false
+    longitude:
+      title: Longitude
+      description: The longitude, in decimal
+      type: string
+      required: false
+  sign_in_location:
+    city:
+      title: City
+      description: City where the sign-in originated. This is calculated using latitude/longitude information from the sign-in activity
+      type: string
+      required: false
+    country_or_region:
+      title: Country Or Region
+      description: Country code info (2 letter code) where the sign-in originated. This is calculated using latitude/longitude information from the sign-in activity
+      type: string
+      required: false
+    geo_coordinates:
+      title: Geo Coordinates
+      description: Geo coordinates
+      type: geo_coordinates
+      required: false
+    state:
+      title: State
+      description: State where the sign-in originated. This is calculated using latitude/longitude information from the sign-in activity
+      type: string
+      required: false
+  risk:
+    id:
+      title: ID
+      description: Unique ID of the risk detection
+      type: string
+      required: true
+    request_id:
+      title: Request ID
+      description: Request ID of the sign-in associated with the risk detection. This property is null if the risk detection is not associated with a sign-in
+      type: string
+      required: false
+    correlation_id:
+      title: Correlation ID
+      description: Correlation ID of the sign-in associated with the risk detection. This property is null if the risk detection is not associated with a sign-in
+      type: string
+      required: false
+    risk_type:
+      title: Risk Type
+      description: The type of risk event detected
+      type: string
+      required: false
+    risk_state:
+      title: Risk State
+      description: The state of a detected risky user or sign-in
+      type: string
+      required: false
+    risk_level:
+      title: Risk Level
+      description: Level of the detected risk
+      type: string
+      required: false
+    risk_detail:
+      title: Risk Detail
+      description: Details of the detected risk. Details for this property are only available for Azure AD Premium P2 customers. P1 customers will be returned hidden
+      type: string
+      required: false
+    source:
+      title: Risk Level
+      description: Source of the risk detection. For example, activeDirectory
+      type: string
+      required: false
+    detection_timing_type:
+      title: Risk Level
```

Comment:
Think this was copy pasted from above should be Detection Timing Type

Reply:
Thanks for catching this. I found another copy pasted error. Will fix both of them.